src/components/Nav/Nav.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+function Nav(state) {
```

Comment:
```suggestion
function Nav(props) {
```
state보다는 props가 의미에 맞아보이고 아직 사용하지 않으니 빈칸으로 두는 것도 좋을 것 같습니다.

Reply:
네! 빈칸으로 두겠습니다.